webpack/scenes/Subscriptions/Manifest/__tests__/SimpleContentAccess.test.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { cleanup } from '@testing-library/react';
+import { renderWithRedux, fireEvent, waitFor } from 'react-testing-lib-wrapper';
```

Comment:
Update these to `patientlyWaitFor` now with https://github.com/Katello/katello/commit/883baf75db86703a3c5efa63baca17e6388c46ea merged